indy_node/server/request_handlers/domain_req_handlers/revoc_reg_def_handler.py

```diff
@@ -36,7 +36,7 @@ def dynamic_validation(self, request: Request):
                                        "Format of {} field is not acceptable. "
                                        "Expected: 'did:marker:signature_type:schema_ref' or "
                                        "'did:marker:signature_type:schema_ref:tag'".format(CRED_DEF_ID))
-        cred_def, _, _, _ = self.get_revoc_reg_def.lookup(cred_def_id, is_committed=False, with_proof=False)
+        cred_def, _, _, _ = self.get_revoc_reg_def.lookup(cred_def_id, is_committed=False, with_proof=True)
```

Comment:
Here we don't require a proof.
A rule: 
- when we return a result to the user (`get_result` in read request handlers) - we do require a proof 
- when we just get the value for some other purposes (validation, checking for existence, etc,) - we do not require a proof